.github/workflows/make-upgrade.yaml

```diff
@@ -0,0 +1,30 @@
+name: Make upgrade
+on: [workflow_dispatch]
+# TODO: schedule this to run weekly or after dependabot update merges
```

Comment:
What is the status of this? I just want this to run every Monday morning ET at 6AM, is that possible?

Reply:
Okay great. Plan was to run it manually the first time, then update with the schedule.

Reply:
Scheduled for 10am UTC (6am ET).